.github/workflows/terraform.yml

```diff
@@ -0,0 +1,44 @@
+name: Terraform Vercel Deployment
+
+on:
+  push:
+    branches:
+      - dev 
+  pull_request:
```

Comment:
this would attempt to apply the terraform changes even if the terraform configuration has not changed.